app/src/main/java/com/ivyclub/contact/ui/main/add_friend/AddFriendFragment.kt

```diff
@@ -1,8 +1,49 @@
 package com.ivyclub.contact.ui.main.add_friend
 
+import android.os.Bundle
+import android.view.View
+import android.widget.ArrayAdapter
+import androidx.fragment.app.viewModels
 import com.ivyclub.contact.R
 import com.ivyclub.contact.databinding.FragmentAddFriendBinding
 import com.ivyclub.contact.util.BaseFragment
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class AddFriendFragment : BaseFragment<FragmentAddFriendBinding>(R.layout.fragment_add_friend) {
+
+    private val viewModel: AddFriendViewModel by viewModels()
+    lateinit var extraInfoListAdapter: ExtraInfoListAdapter
```

Comment:
여기서 바로 선언해줘도 될 것 같은데 따로 lateinit해주신 이유가 있나요?
```suggestion
    private val extraInfoListAdapter = ExtraInfoListAdapter()
```

Reply:
지금 보니 바로 선언하는 게 더 좋을 것 같네요! 수정하겠습니다!